test/integration_test/storkctl_dr_action_test.go

```diff
@@ -286,6 +286,23 @@ func getDestinationKubeConfigFile() (string, error) {
 	return destKubeconfigPath, err
 }
 
+func getSourceKubeConfigFile() (string, error) {
```

Comment:
Let's move this to utils.

Reply:
Done

Reply:
My bad, I meant common_test.go not pkg/utils. Let's keep the common functions used for test here.

Reply:
Done. Please take another look.